.flake8

```diff
@@ -92,8 +89,8 @@ per-file-ignores =
     # F401 imported but unused
```

Comment:
```suggestion
```

Reply:
Is there anything else in that file that can be removed without playing without @arohanajit's work?

If no, you can commit it. I'll be busy soon